src/task_queue/migrations/0003_auto_20220612_2009.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.1.14 on 2022-06-12 20:09
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('task_queue', '0002_auto_20220501_1015'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='task',
+            name='job_name',
+            field=models.CharField(blank=True, choices=[('Super admin nudge', 'Super admin nudge'), ('Community Admin nudge', 'Community Admin nudge')], max_length=100, null=True),
```

Comment:
@abdullai-t - we talked about eliminating the choices spec in models.py.  Can you do that?

Reply:
my bad. Will do that quick